src/renderer/services/api/eth2.ts

```diff
@@ -41,27 +44,39 @@ export class Eth2 implements IBeaconAPIClient {
     }
 
     public async fetchGenesisTime(): Promise<uint64> {
-        return this.httpClient.get<uint64>(FETCH_GENESIS_TIME);
+        return new BN(await this.httpClient.get<string>(FETCH_GENESIS_TIME));
```

Comment:
isn't this BN return type?

Reply:
api will return string as it's only format that can contain big numbers, uin64 = BN